### PR TITLE
v0.10.0 Content (#98)

### LINKED ISSUE #98: Consumables
Usable by any member of the team as a turn in combat. Droppable from combat, purchasable from merchant.

Characterisitics:
- Single use
- Always targets user
- resolves before moves

Ideas:
- AoE damage
- Max hp boost
- Double damage on next attack
- Block

### PR DESCRIPTION
**Pull request by ntseng**
- implemented consumables: Health Potion, Explosion Potion, Smoke Bomb, Repair Kit, Vitamins
- implemented priority queue
- updated Equipment class to EquipmentTemplate
- replaced actionSpeed with priority queue
- italicized flavor texts
- renamed nontargetmove to confirmmove
- renamed targetmove to movetarget
- added vs code settings to gitignore

### CODE REVIEW

**Comment by ntseng** [comment]
Approved by Twitch Chat:tm:

File: Source/Selects/consumable.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+const { getAdventure, checkNextRound, endRound, setAdventure } = require('../adventureDAO');
+const { getConsumable } = require('../consumables/_consumablesDictionary');
+
+const id = "consumable";
+module.exports = new Select(id, async (interaction, [round]) => {
+	// Add a move object to priority moves
+	const adventure = getAdventure(interaction.channelId);
+	if (adventure.room.round === Number(round)) {
+		const [consumableName] = interaction.values;
+		if (consumableName in adventure.consumables && adventure.consumables[consumableName] > 0) {
```
Comment (doubledh) [inline Source/Selects/consumable.js:12]: could probably deny choice of a consumable if the number of outstanding/requested consumables of the party is already equal to the number the party has on-hand.

definitely more particular to code, and more open to race-conditions tho

**Review by doubledh — COMMENTED** [review]
looks good, 1 comment

File: Source/Selects/consumable.js (hunk 1 above)
Comment (ntseng) [reply Source/Selects/consumable.js:12]: So like if someone else is already declared they're gonna use the last Health Potion don't put it in the menu for future potential consumable users?

**Pull request merged**